dds/DCPS/transport/rtps_udp/RtpsUdpReceiveStrategy.cpp

```diff
@@ -115,7 +115,7 @@ namespace {
   {
     if (security_debug.warn) {
       ACE_TCHAR addr_buff[256] = {};
-      remote.addr_to_string(addr_buff, 256, 0);
+      remote.addr_to_string(addr_buff, 256);
```

Comment:
Is this change what we want?  I think the concern is that multiple addresses can map to the same DNS name, so this may be discarding useful info.

Reply:
I was testing in networks where DNS was not working.  In this case, any debug message that prints out an IP address blocks waiting for a timeout.  For the exact reason you stated it is more important to know the IP than it is to resolve the hostname.

Reply:
I must have been confused reading the ACE docs, I think we're agreeing on this one... are there other places that should be changed to make it consistent?